tests/api_connexion/endpoints/test_event_log_endpoint.py

```diff
@@ -233,6 +233,34 @@ def test_should_raises_401_unauthenticated(self, log_model):
 
         assert_401(response)
 
+    def test_should_filter_eventlogs_by_allowed_attributes(self, create_log_model, session):
+        eventlog1 = create_log_model(event="1", dag_id="1", task_id="1", owner="1", when=self.default_time)
+        eventlog2 = create_log_model(event="2", dag_id="2", task_id="2", owner="2", when=self.default_time_2)
```

Comment:
Could you try to use different values? (ex event_1, dag_1, task_1, ...) instead of having the same value for all the variables, without that we cannot ensure that the filter is really working.